docs/atomic-update-container.1.md

```diff
@@ -0,0 +1,34 @@
+% ATOMIC(1) Atomic Man Pages
+% Giuseppe Scrivano
```

Comment:
I was in doubt about that, as atomic update accepts an IMAGE as argument, but yes, I will change it to accept the CONTAINER name when --container is used.  It sounds better than another command.  I would still like to make it clear we are updating a system container, is `atomic update --container --system CONTAINER_NAME` fine?